README.md

```diff
@@ -50,6 +50,8 @@ btcli subnet register --subtensor.network finney --netuid 2 --wallet.name {your_
 
 ### Run the miner
 
+<details>
+<summary>Docker Instructions (not supported during the competition period)</summary>
```

Comment:
We might want to put dates or something here (and below for the validator too)

Reply:
Good idea
Feb 6th - April 20th tentatively (end date subject to change)